server/src/main/java/io/crate/metadata/pgcatalog/OidHash.java

```diff
@@ -63,12 +67,37 @@ static int constraintOid(String relationName, String constraintName, String cons
         return murmurhash3_x86_32(b.bytes, b.offset, b.length, 0);
     }
 
-    public static int functionOid(FunctionName functionName) {
-        BytesRef b = new BytesRef(Type.PROC.toString() + functionName.schema() + functionName.name());
+    public static int regprocOid(FunctionName name) {
+        BytesRef b = new BytesRef(Type.PROC.toString() + name.schema() + name.name());
```

Comment:
The BytesRef creation could be avoided I think. This is probably only doing a utf-16 to utf-8 conversion? Could likely call `.getBytes(...)` on the String directly to get the bytes.

Reply:
does this extend to all the methods in `OidHash`? I mimicked the code.